app/controllers/bikes/recovery_controller.rb

```diff
@@ -12,7 +12,9 @@ def edit
     end
 
     def update
-      if @stolen_record.add_recovery_information(permitted_params)
+      update_params = permitted_params.merge(recovering_user: current_user)
```

Comment:
Use the way this is done on master - it's in the permitted params.

Reply:
View the changes done here: https://github.com/bikeindex/bike_index/pull/846/files